substantial/workflows/utils.py

```diff
@@ -0,0 +1,22 @@
+from datetime import datetime, timezone
+import random
+import uuid
+
+
+class Utils:
+    @staticmethod
+    def now() -> str:
+        return datetime.now(tz=timezone.utc).isoformat()
+
+    @staticmethod
+    def random(a: int, b: int) -> int:
+        return random.randint(a, b)
+
+    @staticmethod
+    def uuid4() -> str:
```

Comment:
why not keep the kind? the user using those should have the same type that usually, he should almost see no difference